### ISSUE TITLE
Support Ruby 3.0

### ISSUE THREAD

**Issue by adsteel** [pusher/push-notifications-ruby#20]
Using Ruby 3.0, `Pusher::PushNotifications.publish` fails:

```ruby
> Pusher::PushNotifications.publish(interests: [])
ArgumentError: wrong number of arguments (given 1, expected 0; required keyword: interests)
```

**Comment by jordansgrant** [comment]
A bit more detail here.

the `caze` gem seems to be the culprit here. Neither the `has_use_case` nor `export` methods properly delegate keyword args resulting in the above error. they should handle `*args`, and `**kwargs`.

I've put a PR up in `caze` to fix the issue and hopefully they will cut a release: https://github.com/magnetis/caze/pull/22

**Comment by elverkilde** [comment]
thanks again @jordansgrant, your PR has been merged and released as 2.0.1
